### PR TITLE
fix MSVC compiler for parse_token

### PR DESCRIPTION
**Pull request by Xavrax**
fix: MSVC compilation error

Fixed compilation error for MSVC in `pubnub_parse_token` function.

### CODE REVIEW

File: core/pubnub_grant_token_api.c [hunk 1]
```diff
@@ -201,7 +201,7 @@ static CborError data_recursion(CborValue* it, int nestingLevel, char** json_res
                         encoded_sig.ptr[0] = '\0';
                     }
 
-                    char base64_str[encoded_sig.size + 2];
+                    char base64_str[67]; // HMAC+SHA256 max size + quotes + tailing null
```
Comment (budgetpreneur) [inline core/pubnub_grant_token_api.c:204]: any reason for hardcoding here with 67?

**Review by budgetpreneur — APPROVED** [review]
I am able to build this PR for grant_token/parse_token example. 👍 
Commented on hardcoded value.

File: core/pubnub_grant_token_api.c (hunk 1 above)
Comment (Xavrax) [reply core/pubnub_grant_token_api.c:204]: According to the [docs](https://www.pubnub.com/docs/sdks/rest-api#grant-access-v3) the `sig` is an HMAC+SHA256 value. Usually we got around 48 bytes but the maximum value of that key can be up to [64 bytes](https://crypto.stackexchange.com/questions/34864/key-size-for-hmac-sha256).

**Comment by mateusz-dahlke** [comment]
@client-engineering-bot release

**Comment by pn-op** [comment]
🚀 Release successfully completed 🚀

**Pull request merged**